sdk/azidentity/CHANGELOG.md

```diff
@@ -0,0 +1,103 @@
+# Release History
```

Comment:
This should have the template at the top with the section that is for the in progress work
```suggestion
# Release History

## v0.10.0-beta.1 (Unreleased)

### Features Added

### Breaking Changes

### Bugs Fixed

### Other Changes
```